docs/source/getting-started/install.rst

```diff
@@ -44,86 +46,82 @@ System Requirements
 Prerequisites
 ^^^^^^^^^^^^^
 
-ScanCode needs a Python 3.6 or a Python 2.7 interpreter.
+ScanCode needs a Python 3.6 (*highly recommended*) or a Python 2.7 interpreter.
 
 .. Note::
 
-    ScanCode currently doesn't support Python 3.7.x, though support will be added soon.
+    ScanCode currently doesn't support Python 3.7.x or 3.8.x, though support will be added soon.
```

Comment:
We should change this to ScanCode currently does not support Python versions above 3.6. 

This way we are covered if a Python 3.9 gets released in the future.